src/references/uniswap/index.ts

```diff
@@ -11,23 +11,20 @@ import { abi as UNISWAP_V2_ROUTER_ABI } from './uniswap-v2-router.json';
 import UNISWAP_V1_EXCHANGE_ABI from './v1-exchange-abi';
 import { RainbowToken } from '@rainbow-me/entities';
 
-const TOKEN_LIST: RainbowToken[] = map(
-  RAINBOW_TOKEN_LIST_DATA.tokens,
-  token => {
-    const { address: rawAddress, decimals, name, symbol, extensions } = token;
-    const address = toLower(rawAddress);
-    return {
-      address,
-      decimals,
-      name,
-      symbol,
-      uniqueId: address,
-      ...extensions,
-    };
-  }
-);
+const tokenList: RainbowToken[] = map(RAINBOW_TOKEN_LIST_DATA.tokens, token => {
+  const { address: rawAddress, decimals, name, symbol, extensions } = token;
+  const address = toLower(rawAddress);
+  return {
+    address,
+    decimals,
+    name,
+    symbol,
+    uniqueId: address,
+    ...extensions,
+  };
+});
 
-const ETHER_WITH_ADDRESS: RainbowToken = {
+const ethWithAddress: RainbowToken = {
```

Comment:
Just as an FYI, I camel cased the constants that were not being exported so it would be easier to distinguish them from others that are actually meant to be exported.